biochemist-python/chapters/Working_with_Pandas.ipynb

```diff
@@ -120,6 +120,7 @@
       "File_Parsing.ipynb\r\n",
```

Comment:
<p>I think we should move this <code>ls</code>  to be indicated in a markdown cell. Because it's in the Jupyter book, it's going to show a lot of thing here that they wouldn't see</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MolSSI-Education/python-scripting-biochemistry/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>